src/discord/commands/parse.ts

```diff
@@ -12,7 +12,9 @@ export default {
     ),
 
   async execute(interaction) {
-    console.log('pdf')
+    console.log('parse')
+    await interaction.deferReply({ ephemeral: true })
```

Comment:
Very nice find! I haven't tested this locally yet (still setting up my environment), but it looks good and makes sense based on what I can find in the `discord.js` API docs

https://discord.js.org/docs/packages/discord.js/main/CommandInteraction:Class#deferReply